src/Sarif.FunctionalTests/v2/ConverterTestData/FxCop/Readme.md

```diff
@@ -0,0 +1,15 @@
+Notes on producing test examples.
+
+This output is an FxCop run against a debug build of SARIF-SDK 
+
+0. Enlist in the SARIF-SDK. You will update the test files in this version.
+1. Git clone into a second version of the tree, e.g.:
+   git clone https://github.com/microsoft/sarif-sdk sarif-sdk-fxcop-test
+2. cd sarif-sdk-fxcop-test. git checkout git checkout fc172277895ed80cb3c03d0ce66738cff89aad75
```

Comment:
>git checkout [](start = 28, length = 12)

These words are duplicated, and I suspect you wanted a line break and a new numbered list item for this command.